users/tests.py

```diff
@@ -1,3 +0,0 @@
-from django.test import TestCase
```

Comment:
Would it make sense to remove these files already in the initial commit that introduced the `user` app to minimize code churn?

Reply:
Good point, I think we can combine the two commits that create the app and add the user model integration.